packs/vsphere/actions/get_vm.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
Yeah this entire pack should use yaml. We can start with this actions onwards and then fix the stragglers later.